File: C_Sharp_Course/C_Sharp_Course/Files/guessing_game.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace C_Sharp_Course
{
    class GussingGame
    {
        static void Guess(string[] args)
        {
            string secretWord = "Course";
            string guess = "";
            int guessCount = 0;
            int guessLimit = 3;
            bool outOfGuesses = false;

            while (guess != secretWord && !outOfGuesses)
            {
                if (guessCount < guessLimit)
                {
                    Console.Write("Enter your guess: ");
                    guess = Console.ReadLine();
                    guessCount++;
                }
                else
                {
                    outOfGuesses = true;
                }
            }

            if (outOfGuesses)
            {
                Console.WriteLine("You lose!");
            }
            else
            {
                Console.WriteLine("You win!");
            }

            Console.WriteLine();

            Console.ReadLine();
        }
    }
}

File: Java_Course/src/App/GuessingGame.java
package App;

import java.util.Scanner;

public class GuessingGame {
	public static void Guessing_Game(String [] args) {
		
		Scanner keyboardInput = new Scanner(System.in);
		
		String secretWord = "Java";
		String guess = "";
		int guessCount = 0;
		int guessLimit = 3;
		boolean outOfGuesses = false;
		
		while(!guess.equals(secretWord) && !outOfGuesses) {
			if(guessCount < guessLimit) {
				System.out.print("Enter a guess: ");
				guess = keyboardInput.nextLine();
				guessCount++;
			} else {
				outOfGuesses = true;
			}
		}
		
		if(outOfGuesses) {
			System.out.println("You lose! You have no more guesses left.");
		} else {
			System.out.println("You win!");
		}
	}
}
